lib/growl.js

```diff
@@ -88,15 +95,11 @@ function display(runner) {
  * @private
  * @callback Growl~growlCB
  * @param {*} err - Error object, or <code>null</code> if successful.
- * @param {string} stdout - <code>stdout</code> from notification delivery
- *     process.
- * @param {string} stderr - <code>stderr</code> from notification delivery
- *     process. It will include timestamp and executed command with arguments.
  */
-function onCompletion(err, stdout, stderr) {
```

Comment:
For what it's worth, I disagree with undocumenting the callback.

Reply:
I was referring to the JSDoc (L91-L94 in original) more so than the function arguments themselves.